packages/webview_flutter/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: webview_flutter
 description: A Flutter plugin that provides a WebView widget on Android and iOS.
-version: 0.3.13
```

Comment:
nit: 0.3.13+1
(I'd consider this a bugfix as it didn't do what the documentation say before._

Reply:
@amirh Done.